lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -334,6 +334,52 @@ class BlockedToMMA : public mlir::RewritePattern {
 };
 } // namespace
 
+static Value promoteOperand(OpBuilder &builder, Location loc, Value operand,
```

Comment:
I noticed that there is another copy of this function in lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp.
This should be avoided because duplication of code is dangerous (the 2 implementation might diverge). @ThomasRaoux could you eliminate one of the copies please and make the remaining function external ?